packages/adblocker-benchmarks/run.js

```diff
@@ -127,6 +128,39 @@ async function memoryUsage(base = { heapUsed: 0, heapTotal: 0, }) {
   return ({ heapUsed, heapTotal, });
 }
 
+async function debug(moduleId, rawLists) {
+  const output = [];
+
+  const Cls = require(moduleId);
+
+  if (Cls.initialize) {
+    await Cls.initialize({ hostsOnly: HOSTS_ONLY });
+  }
+
+  const engine = await Cls.parse(rawLists);
+
+  for (let index = 0; index < requests.length; index += 1) {
+    const { url, frameUrl, cpt } = requests[index];
+
+    if (!isSupportedUrl(url) || !isSupportedUrl(frameUrl)) {
+      continue;
+    }
+
+    const match = engine.match({ type: WEBREQUEST_OPTIONS[cpt], frameUrl, url });
+
+    let matchDebug = null;
+    if (engine.matchDebug) {
```

Comment:
If the engine does not implement `matchDebug()` the default value is `null`. In the end every engine should implement something useful.